demos/imagenet_classification_demo/grid_mat.hpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <set>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include <monitors/presenter.h>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+
+class GridMat {
+public:
+    explicit GridMat(int inputImgsCount,
+                     Presenter& presenter,
+                     const cv::Size maxDisp = cv::Size{1920, 1080},
+                     const cv::Size aspectRatio = cv::Size{16, 9},
+                     double targetFPS = 60
+                     ):
+                     size{std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.height)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.width))
+                                    ),
+                          std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.width)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.height))
+                                    )},
```

Comment:
1. Dependence on batch is no longer needed because there is new image highlighting now.

2. Assuming that gridMat should be completely updated every second:
Then `targetFPS` is the number of frames the grid should contain.
The resulting `size` is `{targetFPS/aspectRatio.height, targetFPS/aspectRatio.width}`. The resulting number of images is `targetFPS/aspectRatio.height * targetFPS/aspectRatio.width == targetFPS^2 / (aspectRatio.height*aspectRatio.width)`. That is equal to `targetFPS` only if `targetFPS == aspectRatio.height*aspectRatio.width`, which is usually false.